src/resources/styles.ts

```diff
@@ -11,6 +11,7 @@ export const darkStyles = {
   "switch-unchecked-button-color": "#999999",
   "switch-unchecked-track-color": "#9b9b9b",
   "divider-color": "rgba(225, 225, 225, .12)",
+  "mdc-button-disabled-ink-color": "rgba(225, 225, 225, .30)",
```

Comment:
```suggestion
  "disabled-text-color": "rgba(225, 225, 225, .30)",
```
And if that doesn't yet, we should add
```
  "mdc-button-disabled-ink-color": "disabled-text-color",
```
to derived styles.